app/src/main/java/com/mathewsachin/fategrandautomata/ui/skill_maker/SkillMakerActivity.kt

```diff
@@ -13,8 +13,11 @@ class SkillMakerActivity : AppCompatActivity(R.layout.skill_maker) {
     val vm: SkillMakerViewModel by viewModels()
     val args: SkillMakerActivityArgs by navArgs()
 
-    fun isMainScreen() = findNavController(R.id.nav_host_fragment_skill_maker)
-        .currentDestination?.id == R.id.main_skill_maker
+    val navHostFragment by lazy {
```

Comment:
The old code crashes the app after the change from fragment to FragmentContainerView and Google tells everyone to use the new code in https://issuetracker.google.com/issues/142847973#comment15

Reply:
Seems to be working just fine.